offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -180,15 +169,32 @@ class OfferToPDFConverter implements OfferExporter {
         List<ProductItem> listOverheadItems = offer.itemsWithOverhead
         List<ProductItem> listNoOverheadItems = offer.itemsWithoutOverhead
 
+        List<ProductItem> listDataGenerationItems = []
+        List<ProductItem> listDataAnalysisItems = []
+        // Sort ProductItems into DataGeneration and Data Analysis
+        listOverheadItems.each {
+            if (it.product instanceof Sequencing) {
+                listDataGenerationItems.add(it)
+            }
+            else {
+                listDataAnalysisItems.add(it)
+            }
```

Comment:
Currently the offer dto and offer entity provide properties that differentiate between overhead and no overhead items and the associated overhead and net cost for each. 
In the list of items with overhead there are 3 dto classes:

1. Sequencing
2. Primary Analysis
3. Secondary analysis

In contrast the list of no overhead items contains the classes 

1.Project Management
2. Data storage

That's why it's enough to sort out the sequencing classes from this list since it only contains item with overheads.

Reply:
the differentiation is a remnant from the previous iteration where it was required to provide subtables for the item grouping into overhead and no overhead association. This was done in the Offer entity and the offer dto by grouping the items into the seperate lists.
Here i just reused that distinction to get to the relevant classes. 
(I think back then the idea was to avoid having business logic in the Converter itself and keep the logic to the Offer entity.)
If you think accessing the classes in the converter makes more sense, then i'll include it in the PR 👍